src/ripple/basics/StringUtilities.h

```diff
@@ -56,25 +74,23 @@ strUnHex(std::size_t strSize, Iterator begin, Iterator end)
 
     if (strSize & 1)
     {
-        int c = charUnHex(*iter);
+        int c = unxtab[*iter++];
 
         if (c < 0)
             return {};
 
         out.push_back(c);
-        ++iter;
     }
 
     while (iter != end)
     {
-        int cHigh = charUnHex(*iter);
+        int cHigh = unxtab[*iter++];
         ++iter;
```

Comment:
By accident you left in a double increment.  Fortunately it causes massive unit test failures.  Notes:
1. You'll need to remove the `++iter;` from this line.
2. Consider running the unit tests before submitting pull requests in the future.  It's easy.  Build rippled, then invoke...
```
$ <path to rippled> --unittest
```
Then wait for the results.  It takes a while, but if it says `0 failures` at the end then you're less likely to have introduced a bug.